src/main/webapp/index2.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
```

Comment:
Quick question: What is the difference between "index2.html" and "index.html"? Is "Index2.html" just for testing the map stuff?